R/analyse_pseudo_bulk.R

```diff
@@ -0,0 +1,373 @@
+#' generate_pseudo_bulk_data 
+#' @param group_by      entry in metadata table, based on these cluster annotation pseudo bulk is performed
+#' @param split_by      variable -> split by a variable within the metadata; k must be a string
+#'                      random -> splits based on a random number; k must be a number
+#'                      Louvain, Louvain_multilevel, SLM, Leiden -> subclusters k must be a list with [resolution, cluster_1, cluster_2]
+#' @param k_variable    variable dependent on the split_by
+#'
+#' @examples
+#' 
+#' #using SCE object
+#' SCE_OBJECT <- ZeiselBrainData()
```

Comment:
Either you load the library first through an explicit `library(...)` our you call the function with the `{LIBRARY NAME}::` decorator.